custom/icds/templates/icds/manage_hosted_ccz.html

```diff
@@ -6,8 +6,6 @@
 {% requirejs_main "icds/js/manage_hosted_ccz" %}
 
 {% block page_content %}
-{% initial_page_data 'appVersionSelectInitialValue' selected_build_details %}
-{% initial_page_data 'appProfileInitialValues' initial_app_profile_details %}
```

Comment:
just confirming that this was not used to pre-fill values in form in case of errors on submit?

Reply:
Another good point. Reverted c58ef7446eba042e304e45b314447d0732ca4ce2

While looking at this, I realized that on prod the version and build profile don't repopulate correctly on form error, although the other fields do. I'm not going to look further into that, but at least this PR shouldn't make things any worse.

Reply:
1. did you forget to push the revert commit or you chose not to revert it?
2. I am confused, you are saying its suppose to work but is not?

Reply:
1. Yep, forgot to push.
2. On prod, it doesn't work fully correctly (version and build profile don't re-populate). The commit that I just reverted would have made it worse (app also would have stopped re-populating).

Reply:
but that is the sole purpose of these variables to repopulate.
anyway not related to this PR and something for me to check out.